security-framework/src/policy.rs

```diff
@@ -0,0 +1,52 @@
+//! Security Policies support.
+use core_foundation::base::TCFType;
+use core_foundation::string::CFString;
+use security_framework_sys::base::{errSecParam, SecPolicyRef};
+use security_framework_sys::policy::*;
+
+use std::mem;
+use std::fmt;
+
+use ErrorNew;
+use base::{Error, Result};
+
+make_wrapper! {
+    /// A type representing a certificate validation policy.
+    struct SecPolicy, SecPolicyRef, SecPolicyGetTypeID
+}
+
+unsafe impl Sync for SecPolicy {}
+unsafe impl Send for SecPolicy {}
+
+impl fmt::Debug for SecPolicy {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        fmt.debug_struct("SecPolicy")
+           .finish()
+    }
+}
+
+impl SecPolicy {
+    /// Creates a `SecPolicy` suitable for validating certificates for SSL.
+    pub fn for_ssl(server: bool, hostname: &str) -> Result<SecPolicy> {
```

Comment:
I try to avoid using `bool` for parameters like this since I can never remember what `SecPolicy::for_ssl(true, "foo.com")` means vs `SecPolicy::for_ssl(false, "foo.com")`. The SecureTransport wrappers use a `ProtocolSide` enum that we could probably use here as well. https://github.com/sfackler/rust-security-framework/blob/master/security-framework/src/secure_transport.rs#L104